### DISCUSSION TITLE
Implementing bullet list

### DISCUSSION THREAD

**Discussion by NikhilSP** [singerdmx/flutter-quill#789] [General]
Hey, I am using this project as guide to implement a bulleted text field in flutter from scratch as for learning practice. I have watched the YouTube videos and spent some time on code but I am not able to locate the code which implements the indentation of text in new line. 

Expected : 
- This is a short message
- This is a long message
   long message continued

What I am getting:
- This is a short message
- This is a long message
long message continued

I want the "long" in the third line to be aligned to "This" in second line. Can you point me to the code where you implement that?

**Comment by singerdmx** [comment]
<img width="1109" alt="image" src="https://user-images.githubusercontent.com/122956/180941505-8d749c30-804e-4d21-b8ac-31f7bd38bd9c.png">

**Reply by EgorK0rshun** [reply]
can I use my characters for lists? For example, squares. Right now the implementation is only for lists with bullets, numbers and checkboxes. What can I override, so that for the ListAttribute(String value) to match my widget to the left of the text in the list?

**Comment by EgorK0rshun** [comment]
can I use my characters for lists? For example, squares. Right now the implementation is only for lists with bullets, numbers and checkboxes. What can I override, so that for the ListAttribute(String value) to match my widget to the left of the text in the list?
